docker-compose.yml

```diff
@@ -55,6 +55,8 @@ services:
       context: ./src
       dockerfile: api/Dockerfile
     container_name: vectorflow_api
+    volumes:
+      - ./src/:/app
```

Comment:
I tested these `volumes` blocks locally on my machine. I originally had to remove them to get the docker compose to work due to the following error:
`python: can't open file '/app/worker/vdb_upload_worker.py': [Errno 1] Operation not permitted`

I was able to resolve this with the recommendation [here](https://stackoverflow.com/questions/58482352/operation-not-permitted-from-docker-container-logged-as-root). We can leave this in for now but I want to flag it that if developers repeatedly run into this issue, we will have to remove it. I will add an update to the README in my current PR to ensure this is documented

Reply:
Okay cool